Source/Config/Frames/Templates/MoneyInput.xml

```diff
@@ -0,0 +1,49 @@
+<Ui xmlns="http://www.blizzard.com/wow/ui/">
+  <!-- Template for money input with label -->
+  <Frame name="AuctionatorConfigurationMoneyInput" mixin="AuctionatorConfigMoneyInputMixin" inherits="AuctionatorConfigurationTooltip" virtual="true">
+    <!--
+      <KeyValues>
+        Number is in copper, but will round to next lowest silver since copper not used in AH
+        <KeyValue key="value" value="0" type="number"/>
+        <KeyValue key="labelText" value="" type="string"/>
+        <KeyValue key="tooltipTitleText" value="" type="string"/>
+        <KeyValue key="tooltipText" value="" type="string"/>
+      </KeyValues>
+    -->
+    <Size y="7"/>
+    <Scripts>
+      <OnLoad method="OnLoad"/>
+    </Scripts>
+    <Anchors>
+      <Anchor point="LEFT" relativeKey="$parent" relativePoint="LEFT"/>
+      <Anchor point="RIGHT" relativeKey="$parent" relativePoint="RIGHT"/>
+    </Anchors>
+    <Frames>
+      <Frame inherits="LargeMoneyInputFrameTemplate" parentKey="MoneyInput">
+        <Size x="150" y="20"/>
+        <KeyValues>
+          <KeyValue key="hideCopper" value="true" type="boolean"/>
```

Comment:
Remove the (I think they're hidden) copper options. Unless we anticipate entering prices in coppers this leaves an untested code path permanently in the codebase.

Sorry, basically criticising this like I'd go through my own code 😦

Reply:
I'm using a predefined Blizz template (`LargeMoneyInputFrameTemplate`), and this parameter is required to hide the coppers in that template. I did not expose this property in the Auctionator template (`AuctionatorConfigurationMoneyInput`) that is used on the configuration panels.